docs/Remove-PSReadlineKeyHandler.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkId=821451
+schema: 2.0.0
+title: Remove-PSReadLineKeyHandler
+---
+
+# Remove-PSReadLineKeyHandler
+
+## SYNOPSIS
+Removes a key binding.
+
+## SYNTAX
+
+```
+Remove-PSReadLineKeyHandler [-Chord] <String[]> [-ViMode <ViMode>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Remove-PSReadLineKeyHandler` cmdlet removes a specified key binding.
+
+## EXAMPLES
+
+### Example 1: Remove a binding
+
+```powershell
+Remove-PSReadLineKeyHandler -Chord Ctrl+Shift+B
+```
+
+This command removes the binding from the key combination, or chord, `Ctrl+Shift+B`.
+The `Ctrl+Shift+B` chord is created in the `Set-PSReadLineKeyHandler` article.
+
+## PARAMETERS
+
+### -Chord
+
+Specifies an array of keys or sequences of keys to be removed. A single binding is specified by
+using a single string. If the binding is a sequence of keys, separate the keys by a comma, as in
+the following example:
+
+`Ctrl+X,Ctrl+L`
```

Comment:
See previous comment about `Shift` - this example should probably be `Ctrl+x,Ctrl+l`.

Reply:
Will fix